intel_extension_for_transformers/neural_chat/utils/mpt_trace.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2023 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import torch
+from typing import Optional, Tuple
+from transformers.modeling_outputs import CausalLMOutputWithPast
+from optimum.intel.generation.modeling import TSModelForCausalLM # pylint: disable=E0401
+
+
+def jit_trace_mpt_7b(model):
```

Comment:
Could you make this function more generalizable  as the utility? for example for ModelForCausalLM.